backend/test/test_no_waste.py

```diff
@@ -0,0 +1,34 @@
+from no_waste import no_waste
+from no_waste.no_waste import Recipe, Product
+
+
+import pytest
+
+@pytest.fixture
+def client():
+    return no_waste.app.test_client()
+
+
+def test_nonempty_initial_db():
+    assert Recipe.query.count() != 0
```

Comment:
Each test should start with a clean state.

Reply:
There is an initial database state and this is what this test tests